Makefile.rules

```diff
@@ -15,9 +15,6 @@ endif
 # Otherwise, print the full command line.
 quiet ?= $($1)
 
-%.html: %.rst
-	rst2html --strict --no-raw $< > $@
```

Comment:
This is being added back in the following commit. I understand the rationale, so if you want this in the history for future reference it's fine with me, just wanted to make sure it wasn't a mistake.

Reply:
Confused. Where is this being added back?

Reply:
Oh man... I _think_ what happened was that I clicked on the referenced commits to have a look and then proceeded to consider one of them part of the PR :man_facepalming:

Ignore me -.-

Reply:
Happens to the best of us! :rofl: